source/geometries/SiPMpetVUV.cc

```diff
@@ -36,19 +34,28 @@ namespace nexus {
 			    visibility_(0),
 			    refr_index_(1),
```

Comment:
Since the quartz refraction index is ~1.6, I would either change this default value or either modify the macro with the reasonable value of this property so that when someone executes the macro in the examples, they get a value closer to the true one.

Reply:
I did the former.